Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -288,6 +288,9 @@ extension Module  {
         // At the moment, set the Swift version to 3 (we will need to make this dynamic), but for now this is necessary.
         buildSettings["SWIFT_VERSION"] = "3.0"
         
+        // Defined for regular `swift build` instantiations, so also should be defined here.
+        buildSettings["OTHER_SWIFT_FLAGS"] = "-DSWIFT_PACKAGE"
```

Comment:
I think this should use `SWIFT_ACTIVE_COMPILATION_CONDITIONS` instead of `OTHER_SWIFT_FLAGS`, otherwise LGTM.